extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -109,11 +144,42 @@ export class AmpSelector extends AMP.BaseElement {
         this.clearSelection_(element);
       }
     }
+    this.updateFocus_();
     // Update inputs.
     this.setInputs_();
   }
 
   /**
+   * Determine which option should receive focus first and set tabIndex
+   * on all options accordingly.
+   * In multi-select selectors, focus should just go to the first option.
+   * In single-select selectors, focus should go to the initially selected
+   * option, or to the first option if none are initially selected.
+   * @private
+   */
+  updateFocus_() {
+    let firstSelectedIndex = -1;
```

Comment:
this method should only be called if mode != none. Can't find any guard against that.